docs/content/en/blog/news/november-2021-update.md

```diff
@@ -0,0 +1,178 @@
+  with:
+    duration: 30m
+    metrics:
+    - strategy: CANARY_BASELINE
+      provider: my-prometheus
+      deviation: HIGH
+      interval: 5m
+      query: sum (rate(http_requests_total{job="foo-{{ .Variant.Name }}", status=~"5.*"}[5m])) / sum (rate(http_requests_total{job="foo-{{ .Variant.Name }}"}[5m]))
+{{< /tab >}}
+{{< tab lang="yaml" header="Using THRESHOLD strategy" >}}
+# ADA by checking the given queries againts the given thresholds.
+- name: ANALYSIS
+  with:
+    duration: 30m
+    metrics:
+    - strategy: THRESHOLD
+      provider: my-prometheus
+      interval: 5m
+      expected:
+        max: 0.01
+      query: sum (rate(http_requests_total{status=~"5.*"}[5m])) / sum (rate(http_requests_total[5m]))
+{{< /tab >}}
+{{< /tabpane >}}
+
+Check out its [documentation](/docs/user-guide/automated-deployment-analysis/#strategies) for more details.
+
+##### Notification
+
+Any PipeCD event (deployment triggered, planned, analysis completed...) can be configured to send to external services like Slack or a Webhook service. While forwarding those events to a chat service helps developers have a quick and convenient way to know the deployment’s current status, forwarding to a Webhook service may be useful for triggering other related tasks.
+
+Until now, only Slack has been supported. From `v0.21.0` you can configure your Pipeds to send its events to external services via Webhook. See [here](http://localhost:1313/docs/operator-manual/piped/configuring-notifications/#sending-notifications-to-external-services-via-webhook) to know how to configure.
+
+Besides that, events sent to Slack can be configured to mention one or multiple Slack users. It would be helpful to prevent developers from missing important events.
+
+``` yaml
+apiVersion: pipecd.dev/v1beta1
+kind: KubernetesApp
+spec:
+  pipeline:
+    stages:
+      - name: K8S_CANARY_ROLLOUT
+      - name: WAIT_APPROVAL
+      - name: K8S_PRIMARY_ROLLOUT
+      - name: K8S_CANARY_CLEAN
+  notification:
+    mentions:
+      - event: DEPLOYMENT_WAIT_APPROVAL
+        slack:
+          - slack-user-id-1
+          - slack-user-id-2
+      - event: "*"                      # Specifying "*" means mentioning the given users for all events.
+        slack:
+          - slack-user-id-3
+```
+
+##### Piped Operation
+
+As you know, PipeCD is designed with `control-plane` and `agent` model (see [component architecture diagram](/docs/overview/)). Piped is a single binary agent you run in your cluster, your local network to handle the deployment tasks related to that cluster. It can be run as a Pod inside a Kubernetes cluster, as a process in a virtual machine or even on your local machine. Piped is stateless so that it can be restarted without worrying about data loss. That helps the product team reduce the cost of maintaining their Pipeds. You can see these [installation guides](https://pipecd.dev/docs/operator-manual/piped/installation/) to know how to install Piped on each environment.
```

Comment:
```suggestion
As you know, PipeCD is designed with `control-plane` and `agent` model (see [component architecture diagram](/docs/overview/)). Piped is a single binary agent you run in your cluster, your local network to handle the deployment tasks related to that cluster. It can be run as a Pod inside a Kubernetes cluster, as a process in a virtual machine, or even on your local machine. Piped is stateless so that it can be restarted without worrying about data loss. That helps the product team reduce the cost of maintaining their Pipeds. You can see those [installation guides](https://pipecd.dev/docs/operator-manual/piped/installation/) to know how to install Piped on each environment.
```
nits